waku/v2/protocol/relay/waku_relay_test.go

```diff
@@ -302,3 +305,53 @@ func TestWakuRelayAutoShard(t *testing.T) {
 	require.NoError(t, err)
 
 }
+
+func TestInvalidPushMessages(t *testing.T) {
```

Comment:
Minor one: rename to `TestInvalidMessagePublish`

Reply:
Done at https://github.com/waku-org/go-waku/pull/916/commits/c076e36cec70d43bf80266b6ae33b175ce655860